bazel/repository_locations.bzl

```diff
@@ -1247,6 +1248,7 @@ REPOSITORY_LOCATIONS_SPEC = dict(
             "envoy.filters.network.wasm",
             "envoy.stat_sinks.wasm",
             "envoy.rbac.matchers.upstream_ip_port",
+            "envoy.rbac.uri_template.uri_template_matcher",
```

Comment:
I don't get why this is needed?

Reply:
I added it because of an [error message](https://github.com/envoyproxy/envoy/actions/runs/7612248977/job/20729480652#step:8:628) in Envoy/Prechecks :
```
Dependency validation failed, please check metadata in bazel/repository_locations.bzl
Extension envoy.rbac.matchers.uri_template depends on com_github_google_flatbuffers but com_github_google_flatbuffers does not list envoy.rbac.matchers.uri_template in its allowlist
```
However, it seems to me as if the checks are bugged and checkout older code, because when I run it locally I don't get this error know - I think `envoy.rbac.matchers.uri_template` value comes from [source/extensions/extensions_build_config.bzl](https://github.com/envoyproxy/envoy/pull/31447/files/e1acfe4a996cce7813264e69efed8424ce7366e7#diff-5294c859ad138c7624de53e0f9566c0b630f5a9522ff357d23b2f0b17d6e3991)